lib/grpcServer/handlers/platform/getConsensusParamsHandlerFactory.js

```diff
@@ -0,0 +1,75 @@
+const InvalidArgumentGrpcError = require('@dashevo/grpc-common/lib/server/error/InvalidArgumentGrpcError');
+
+const {
+  server: {
+    error: {
+      InternalGrpcError,
+    },
+  },
+} = require('@dashevo/grpc-common');
+
+const {
+  v0: {
+    GetConsensusParamsResponse,
+    ConsensusParamsBlock,
+    ConsensusParamsEvidence,
+  },
+} = require('@dashevo/dapi-grpc');
+
+const RPCError = require('../../../rpcServer/RPCError');
+
+/**
+ *
+ * @param {getConsensusParams} getConsensusParams
+ * @returns {getConsensusParamsHandler}
+ */
+function getConsensusParamsHandlerFactory(getConsensusParams) {
+  /**
+   * @typedef getConsensusParamsHandler
+   * @param {Object} call
+   * @returns {Promise<>}
+   */
+  async function getConsensusParamsHandler(call) {
+    const { request } = call;
+
+    const prove = request.getProve();
+
+    if (prove) {
+      throw new InvalidArgumentGrpcError('Prove is not implemented yet');
+    }
+
+    let consensusParams;
+
+    try {
+      consensusParams = await getConsensusParams();
+    } catch (e) {
+      if (e instanceof RPCError) {
+        throw new InternalGrpcError(e);
+      }
+
+      throw e;
```

Comment:
I guess it could respond with different errors, for example the height is not valid or something like that. In this case you nee to you other gRPC error codes.